### PR TITLE
PLAT-1288: Fix popover and calendar tag prefix

### PR DESCRIPTION
**Pull request by mihaelpalfi**
[JIRA ticket](https://reciprocitylabs.atlassian.net/browse/PLAT-1288)

#### Description
As per JIRA ticket

#### ChangeLOG

[Changelog](https://zen-ui.zengrc.com/?path=/docs/changelog--page)

#### Useful links
[Zen UI readme](https://github.com/reciprocity/zen-ui/blob/main/README.md)

[PR review guidelines](https://github.com/reciprocity/zengrc/blob/develop/doc/dev_environment_and_process/pull_request_reviews.md)

[Git commit guidelines](https://github.com/reciprocity/zengrc/blob/develop/doc/dev_environment_and_process/commit_guidelines.md)

### CODE REVIEW

File: src/components/zen-date-picker/zen-date-picker.tsx [hunk 1]
```diff
@@ -97,47 +97,55 @@ export class ZenDatePicker {
...
-        <zen-input placeholder={this.placeholder} value={this.formattedDate}>
-          <zen-space padding="md none md md" slot="leadingSlot">
-            <zen-icon icon={faCalendarAlt}></zen-icon>
-          </zen-space>
-        </zen-input>
+        <sb-zen-input placeholder={this.placeholder} value={this.formattedDate}>
+          <sb-zen-space padding="md none md md" slot="leadingSlot">
+            <sb-zen-icon icon={faCalendarAlt}></sb-zen-icon>
+          </sb-zen-space>
+        </sb-zen-input>
```
Comment (f-jauregui) [inline src/components/zen-date-picker/zen-date-picker.tsx:104]: Inside the component is a bit different than the stories.
We need to create the dynamic tags so we support any incoming prefix, not just `sb`.
In order to do that:

```js
const ZenInput = applyPrefix('zen-input', this.hostElement);
const ZenSpace = applyPrefix('zen-space', this.hostElement);
const ZenIcon = applyPrefix('zen-icon', this.hostElement);

<ZenInput placeholder={this.placeholder} value={this.formattedDate}>
  <ZenSpace padding="md none md md" slot="leadingSlot">
    <ZenIcon icon={faCalendarAlt}></ZenIcon>
  </ZenSpace>
</ZenInput>
```

Do that for this and the popover component.

**Review by f-jauregui — CHANGES_REQUESTED** [review]
Just those little changes in the component main files.

File: src/components/zen-date-picker/zen-date-picker.tsx (hunk 1 above)
Comment (mihaelpalfi) [reply src/components/zen-date-picker/zen-date-picker.tsx]: Done.

**Comment by chavlji** [comment]
Was broken before. This pr fixes it!

**Review by chavlji — APPROVED** [review]
Was broken before. This pr fixes it!

**Review by chavlji — APPROVED** [review]

**Comment by chavlji** [comment]
@f-jauregui I'm merging this because it's blocking me. Please check it out and we can open a new pr if there are any flaws.

**Pull request merged**